provisioning/api/handler.go

```diff
@@ -50,6 +54,49 @@ const (
 	ProvisioningSessionMediaType = "application/vnd.veraison.provisioning-session+json"
 )
 
+func (o *Handler) GetServiceState(c *gin.Context) {
+	state := proto.ServiceState{
+		ServerVersion: "0.0.1",
```

Comment:
Should the Server Version be hard coded or should it be set from a Config file? Just a question:

Reply:
I think version should be computed at build time from git metadata (tags, raw commit info) and assigned to a predefined global via `go build -w -X VERSION_VAR=VERSION_VAL`

Reply:
Raised #51 to track this.

Reply:
No need :). Just pushed an update that sets versions from git.

Reply:
@setrofim Too cool 👍